data/gegno/gegno ships.txt

```diff
@@ -0,0 +1,1135 @@
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# Gegno:
+
+ship Shale
+	sprite ship/shale
+	thumbnail thumbnail/shale
+	swizzle 0
+	attributes
+		category Transport
+		licenses
+			"Gegno Civilian"
+		cost 265000
+		mass 121
+		bunks 7
+		"cargo space" 36
+		drag 1.8
+		"engine capacity" 58
+		"fuel capacity" 400
+		"heat dissipation" 0.8
+		"heat protection" 3
+		"ion protection" 3
+		hull 760
+		"outfit space" 113
+		"required crew" 1
+		shields 0
+		"weapon capacity" 0
+		weapon
+			"blast radius" 18
+			"shield damage" 40
+			"hull damage" 132
+			"hit force" 210
+	outfits
+		"C3 Scrap Cell"
+		Hyperdrive
+		"R01 Skirmish Battery"
+		"RG15 Torch Steering"
+		"RG15 Torch Thruster"
+	engine 0 40
+	leak leak 50 60
+	explode "small explosion" 5
+	explode "tiny explosion" 10
+	description "Due to the Gegno splitting up and colonizing new worlds, they needed a smaller ship to do a lot of the grunt work instead of always relying on their enormous (and expensive) City-Ships. The Shale is the Gegno's equivalent of a shuttle: small and compact with transportation being its single purpose. Whether it be cargo, trade goods, travellers, or recruits, Shales are the foundation of the Gegno interstellar civilization."
+
+ship Dolomite
+	sprite ship/dolomite
+	thumbnail thumbnail/dolomite
+	swizzle 0
+	attributes
+		category "Light Freighter"
+		licenses
+			"Gegno Civilian"
+		cost 1692000
+		mass 570
+		bunks 80
+		"cargo space" 155
+		drag 5.9
+		"engine capacity" 81
+		"fuel capacity" 500
+		"heat dissipation" 0.7
+		"heat protection" 3
+		"ion protection" 3
+		hull 10500
+		"outfit space" 150
+		"required crew" 34
+		shields 0
+		"weapon capacity" 0
+		weapon
+			"blast radius" 72
+			"shield damage" 320
+			"hull damage" 620
+			"hit force" 720
+	outfits
+		"C3 Scrap Cell"
+		Hyperdrive
+		"R01 Skirmish Battery"
+		"RG15 Torch Steering" 2
+		"RG15 Torch Thruster"
+	engine -21 155
+	engine 21 155
+	leak leak 60 50
+	leak flame 60 80
+	explode "large explosion" 14
+	explode "medium explosion" 24
+	explode "small explosion" 26
+	explode "tiny explosion" 10
+	"final explode" "final explosion medium"
+	description "Almost as large as a human cruiser, Dolomites are the Gegno's most common jack-of-all-trades cargo vessel in service, having been used since the early days of space colonization. They are used as both passenger liners and cargo vessels, and carry decent amounts of both to note. Due to its popularity and continuous usage, both the Gegno Vi and Gegno Scin factions have developed further on the original design for their own purposes."
```

Comment:
```suggestion
	description "Almost as large as a human cruiser, Dolomites are the Gegno's most common cargo vessel in service, having been used since the early days of space colonization. They are used as both passenger liners and cargo vessels. Due to its popularity and continuous usage, both the Gegno Vi and Gegno Scin factions have developed upon on the original design for their own purposes."
```